### DISCUSSION TITLE
Kandinsky 2.1 support (NO POLITICS)  -

### DISCUSSION THREAD

**Discussion by adrianpuiu** [vladmandic/sdnext#293] [General]
Vlad, getting back on the subject from https://github.com/vladmandic/automatic/discussions/263 

Please let me know and lock the discussion after.

**Comment by vladmandic** [comment]
honestly, it does look interesting and i'd be for integrating it (regardless of politics).
the issue is my current workload, so i doubt it would happen anytime soon.
i'd welcome any contribution on that topic.

**Reply by adrianpuiu** [reply]
I already have a small gradio app for it but only for txt2image... dreambooth and textual inversion scripts to( found em them in a kandinsky fork ) ... i'd like to give it a shot but don't know what file(s) to look at

**Comment by myndxero** [comment]
Sorry :<

**Comment by kohweiteck** [comment]
I also interested in Karlo as it uses same foundation same as the OpenAI diffusion tech.

**Comment by user425846** [comment]
Is anyone able to integrate this? I would be very interested in using it

**Comment by 0-NiK-0** [comment]
Kandinsky 2.1
https://github.com/ai-forever/Kandinsky-2
https://github.com/Blucknote/Kandinsky-advanced-notebooks
https://huggingface.co/ai-forever/Kandinsky_2.1
https://huggingface.co/spaces/ai-forever/Kandinsky2.1

Demo
https://fusionbrain.ai/
https://t.me/kandinsky21_bot
https://rudalle.ru/kandinsky2

Info
https://habr.com/ru/companies/sberbank/articles/701162/
https://habr.com/ru/companies/sberbank/articles/725282/

**Comment by adrianpuiu** [comment]
it seems someone else already did a better job then mine : https://github.com/seruva19/kubin.git
